_data/students.yaml

```diff
@@ -131,7 +131,12 @@ undergrad_students:
   - first_name: Samuel
     last_name: VanDenBerghe
     image: Samuel_VanDenBerghe.jpg
+    
   - first_name: Ramez
     last_name: Gammoh
     image: Ramez_Gammoh.jpg
     link: https://ramez-git.github.io/Ramez.github.io/
+    
+    - first_name: Grace
+    last_name: Walker
+    image: Grace_Walker.jpg
```

Comment:
I think you need to fix the indentation here in order for it to build correctly.  Thanks!